bluebrain/repo-bluebrain/packages/brayns/package.py

```diff
@@ -31,12 +31,18 @@ class Brayns(CMakePackage):
     depends_on("rkcommon@1.10.0")
     depends_on("ospray@2.10.6")
 
-    depends_on("spdlog@1.9.2:")
+    depends_on("spdlog@1.9.2:", when="3.8.3")
     depends_on("poco@1.12.4")
-    depends_on("glm")
+    depends_on("glm", when="3.8.3")
```

Comment:
can we add a range here? or just 3.8.3 requires this? Just thinking for the future

Reply:
ok